File: Hrms.Web/src/app/admin/directory/models/team.model.ts
export class TeamModel {
    id: number;
    employeeId: number;
    managerId: number;
    type: number;
    department: string;
    designation: string;
    managerName: string;
    employeeName: string;
    canEdit: boolean;
    typeName: string;
    canDirectEdit: boolean;
}

File: Hrms.Core/Models/Employee/TeamModel.cs
using Hrms.Core.Utilities;

namespace Hrms.Core.Models.Employee
{
    public class TeamModel
    {
        public int Id { get; set; }
        public int EmployeeId { get; set; }
        public string EmployeeName { get; set; }
        public int? ManagerId { get; set; }
        public string ManagerName { get; set; }
        public string Department { get; set; }
        public string Designation { get; set; }
        public int Type { get; set; }
        public Constants.RecordStatus Status { get; set; }
    }
}
